commcare_connect/organization/tests/test_forms.py

```
@@ -223,3 +258,41 @@ def test_validation_error_mismatched_llo_entity(self):
         ]
+
+    def test_create_new_llo_entity_with_short_name(self):
+        form = OrganizationSelectOrCreateForm(
+            data={
+                "org": TOMSELECT_NEW_ENTRY_PREFIX + "New Org",
+                "llo_entity": TOMSELECT_NEW_ENTRY_PREFIX + "New LLO",
+                "llo_entity_short_name": "NL",
+            }
+        )
+        assert form.is_valid(), form.errors
+        org, is_new_org = form.save()
```

Ruff (0.15.7)

[warning] Unpacked variable `is_new_org` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)